src/main/kotlin/domain/person/Dealer.kt

```diff
@@ -1,26 +1,14 @@
 package domain.person
 
 import domain.card.Card
-import domain.constant.BlackJackConstants.BLACK_JACK
-import domain.constant.BlackJackConstants.DEALER_STAND_CONDITION
-import domain.constant.GameState
-import domain.constant.GameState.BUST
-import domain.constant.GameState.HIT
-import domain.constant.GameState.STAND
-import domain.result.CardsScore
+import domain.card.Hand
+import domain.state.State
+import domain.state.dealerState.DealerFirstTurn
 
-class Dealer(override val name: String = "딜러") : Person(name) {
-    fun showOneCard(): List<Card> {
-        return cards.value.subList(0, 1)
-    }
+class Dealer(name: String = "딜러") : Person(name) {
 
-    override fun checkState(): GameState {
-        val totalNumber = CardsScore.getTotalCardNumber(cards)
-        return when {
-            totalNumber > BLACK_JACK -> BUST
-            totalNumber > DEALER_STAND_CONDITION -> STAND
-            totalNumber <= BLACK_JACK -> HIT
-            else -> throw IllegalStateException()
-        }
+    override var state: State = DealerFirstTurn(Hand())
+    fun showOneCard(): List<Card> {
```

Comment:
```suggestion
    override var state: State = DealerFirstTurn(Hand())
    
    fun showOneCard(): List<Card> {
```
변수와 함수 사이는 개행으로 구분해주는 것이 좋습니다.